src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,49 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migrating from `io.sentry:sentry-android 6.x` to `io.sentry:sentry-android 7.0.0`
+
+### Breaking Changes
+
+- The minimum supported API level was updated to 19 (`minSdk`).
+- The `sentry-android-okhttp` classes were deprecated in favor of `sentry-okhttp`, which is a pure Java module and can be used in non-Android projects.
+  - The `SentryOkHttpUtils` class was removed from the public API. If you were using it, consider filing a [feature request](https://github.com/getsentry/sentry-java/issues).
+- If you're using `sentry-kotlin-extensions`, it now requires `kotlinx-coroutines-core` version `1.6.1` or higher.
+- Moved `enableNdk` and `enableScopeSync` from `SentryOptions` to `SentryAndroidOptions`.
+  - Changed the return type of `SentryApolloInterceptor.BeforeSpanCallback` from `ISpan` to `ISpan?`.
+
+### Behavioral Changes
+
+- `Sentry.getSpan()` now returns the root span/transaction instead of the latest span.
```

Comment:
That's only on Android right?
```suggestion
- `Android Only: Sentry.getSpan()` now returns the root span/transaction instead of the latest span.
```

Reply:
yeah, but it's android migration docs, so we're good. I'll open a separate PR for the Java migration docs